### PR TITLE
Update winit to 0.28

### PR DESCRIPTION
**Pull request by emilk**
I also removed `Cargo.lock` from git, since this is a library (see https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html). Let me know if I went to far here!

### CODE REVIEW

**Comment by emilk** [comment]
Sure, no problem!

File: platforms/winit/Cargo.toml [hunk 1]
```diff
@@ -23,6 +27,3 @@ accesskit_macos = { version = "0.4.2", path = "../macos" }
 
 [target.'cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "openbsd", target_os = "netbsd"))'.dependencies]
 accesskit_unix = { version = "0.1.1", path = "../unix" }
-
-[dev-dependencies]
-winit = "0.27.2"
```
Comment (emilk) [inline platforms/winit/Cargo.toml:28]: It is already a proper dependency

**Comment by mwcampbell** [comment]
The reason I listed winit again under dev-dependencies was so our example could run with all default features, including Wayland client-side decorations where necessary. Disabling that feature in the base dependency on winit is no longer as necessary, now that winit doesn't depend on crossfont by default, but I'd rather keep it that way.

**Comment by emilk** [comment]
Well, serves me right for thinking I know better :)

**Comment by mwcampbell** [comment]
No worries. You _are_ a more experienced Rust dev than I am. Guess I should have added a comment there.

**Comment by mwcampbell** [comment]
OK, accesskit_winit v0.9.0 is published. I realize this is necessary to unblock the update to winit 0.28 in egui.

**Comment by emilk** [comment]
Thanks for the quick response! 🙏

**Pull request merged**